File: library/src/main/java/com/github/dmitrikudrenko/countryphonecodechooser/ui/adapter/OnCodeSelectListener.java
package com.github.dmitrikudrenko.countryphonecodechooser.ui.adapter;

import android.support.annotation.NonNull;
import android.support.annotation.RestrictTo;
import android.support.annotation.RestrictTo.Scope;
import com.github.dmitrikudrenko.countryphonecodechooser.model.CountryCode;

@RestrictTo(Scope.LIBRARY)
public interface OnCodeSelectListener {
    void onCodeSelected(@NonNull CountryCode countryCode);
}

File: library/src/main/kotlin/com/github/dmitrikudrenko/countryphonecodechooser/ui/adapter/OnCodeSelectListener.kt
package com.github.dmitrikudrenko.countryphonecodechooser.ui.adapter

import android.support.annotation.RestrictTo
import android.support.annotation.RestrictTo.Scope
import com.github.dmitrikudrenko.countryphonecodechooser.model.CountryCode

@RestrictTo(Scope.LIBRARY)
interface OnCodeSelectListener {
    fun onCodeSelected(countryCode: CountryCode)
}
